yolapy/models/user.py

```diff
@@ -0,0 +1,54 @@
+from yoconfig import get_config
+from yolapy.services import Yola
+
+
+class User(object):
+
+    """Yola User."""
+
+    def __init__(self, active=False, deleted=None, email=None, id=None,
+                 name=None, partner_id=None, preferences=None,
+                 signup_date=None, surname=None):
+        """Sets all user attributes and inits a Yola client."""
+        self._partner_id = partner_id
+        self.active = active
+        self.deleted = deleted
+        self.email = email
+        self.id = id
+        self.name = name
+        self.preferences = preferences or {}
+        self.signup_date = signup_date
+        self.surname = surname
+
+        self.client = Yola()
+
+    @property
+    def partner_id(self):
+        """Use the auth user of the Yola client as the default partner ID."""
+        return self._partner_id or get_config('yola')['auth'][0]
+
+    @partner_id.setter
+    def partner_id(self, val):
+        self._partner_id = val
+
+    def _save_create(self):
+        user = self.client.create_user(
+            email=self.email,
+            name=self.name,
+            partner_id=self.partner_id,
+            preferences=self.preferences,
+            surname=self.surname)
+        self.update(**user)
+
+    def _save_update(self):
+        raise NotImplementedError()
+
+    def save(self):
+        if self.id:
+            self._save_update()
+        else:
+            self._save_create()
```

Comment:
I'm unsure about this, it is a slight "tell don't ask" violation.

I'm wondering if it would be better to implement a PUT method in the client vs using create/update?